src/web/screens/clientSideIntegration.tsx

```diff
@@ -35,13 +46,38 @@ function ClientSideIntegration() {
     }
   };
 
+  const handleUpdateDomainNames = async (updatedDomainNames: string[]) => {
+    try {
+      const response = await SetDomainNames(updatedDomainNames);
```

Comment:
These two functions only differing by first letter case makes it easy to misunderstand what's going on here. Could you find a different name for one of them?

Reply:
fixed!